File: Data Structures & Algorithms/custom-sort-string/submission-1.py
class Solution:
    def customSortString(self, order: str, s: str) -> str:
        output = ""
        the_dict = {}
        remain = ""
        for i in range(len(s)):
            the_dict[s[i]] = the_dict.get(s[i] , 0)+1
        for i in order:
            if i in the_dict:
                output += i * the_dict[i]
                del the_dict[i]
        for key, value in the_dict.items():
            output += key * value

        return output

File: Data Structures & Algorithms/replace-elements-with-greatest-element-on-right-side/submission-1.java
class Solution {
    public int[] replaceElements(int[] arr) {
        int[] output = new int[arr.length];
        for (int i = 0; i < arr.length-1; i++)
        {
            int currentmax = 0;
            for (int j = i + 1; j < arr.length;j++)
            {
                if (arr[j] > currentmax)
                {
                    currentmax = arr[j];
                }
            }
            output[i] = currentmax;
        }
        
        output[arr.length-1] = -1;

        return output;
    }
}